src/Test/Perf/util/test_util.csx

```diff
@@ -200,6 +216,41 @@ ProcessResult ShellOut(
     };
 }
 
+ProcessResult ShellOutUsingShellExecute(
```

Comment:
oh ok.

Today `runner.csx` will download CPC before running the tests. But I will later introduce an option to make it depend on a switch. If it is run as part of the `automation.csx` then run download else, it wont run. These will be incremental changes and I have a Task List at #9951 to track these changes